lib/wallet.js

```diff
@@ -175,6 +175,8 @@ function authorizeZeroConf (settings, tx, machineId) {
 
   if (plugin === 'all-zero-conf') return Promise.resolve(true)
 
+  if (plugin === 'none') return Promise.resolve(false)
```

Comment:
I don't think this is correct, if no zeroConf plugin is configured and tx.fiat < zeroConfLimit why would we wait for confirmation?

Also we don't use 'all-zero-conf' anymore, so I think this PR can be renaming 'all-zero-conf' to 'none'

Reply:
Ah I misunderstood `tx.fiat.gt(zeroConfLimit)`... If there's no plugin, then the tx should by always accepted, no matter the limit, no? That's what I thought initially.